extensions/wikia/ArticleVideo/scripts/ooyala/skin.json

```diff
@@ -0,0 +1,265 @@
+        "languageFile": "//player.ooyala.com/static/v4/stable/4.6.9/skin-plugin/zh.json",
+        "androidResource": "skin-config/zh.json",
+        "iosResource": "zh"
+      }
+    ]
+  },
+  "responsive": {
+    "breakpoints": {
+      "xs": {"id": "xs", "name": "oo-xsmall", "maxWidth": 559, "multiplier": 0.7},
+      "sm": {"id": "sm", "name": "oo-small", "minWidth": 560, "maxWidth": 839, "multiplier": 1},
+      "md": {"id": "md", "name": "oo-medium", "minWidth": 840, "maxWidth": 1279, "multiplier": 1},
+      "lg": {"id": "lg", "name": "oo-large", "minWidth": 1280, "multiplier": 1.2}
+    },
+    "aspectRatio": "auto"
+  },
+  "startScreen": {
+    "promoImageSize": "default",
+    "showPlayButton": true,
+    "playButtonPosition": "center",
+    "playIconStyle": {
+      "opacity": 1
+    },
+    "showTitle": true,
+    "showDescription": false,
+    "titleFont": {
+      "color": "white"
+    },
+    "descriptionFont": {
+      "color": "white"
+    },
+    "infoPanelPosition": "topLeft",
+    "showPromo": true
+  },
+  "pauseScreen": {
+    "showPauseIcon": true,
+    "pauseIconPosition": "center",
+    "PauseIconStyle": {
+      "color": "white",
+      "opacity": 1
+    },
+    "showTitle": true,
+    "showDescription": false,
+    "infoPanelPosition": "topLeft",
+    "screenToShowOnPause": "default"
+  },
+  "endScreen": {
+    "screenToShowOnEnd": "discovery",
+    "showReplayButton": true,
+    "replayIconStyle": {
+      "color": "white",
+      "opacity": 1
+    },
+    "showTitle": false,
+    "showDescription": false,
+    "infoPanelPosition": "topLeft"
+  },
+  "adScreen": {
+    "showAdMarquee": true,
+    "showAdCountDown": true,
+    "showControlBar": false
+  },
+  "discoveryScreen": {
+    "panelTitle": {
+      "titleFont": {
+        "fontSize": 28,
+        "fontFamily": "Roboto Condensed",
+        "color": "white"
+      }
+    },
+    "contentTitle": {
+      "show": true,
+      "font": {
+        "fontSize": 22,
+        "fontFamily": "Roboto Condensed",
+        "color": "white"
+      }
+    },
+    "contentDuration": {
+      "show": true,
+      "font": {
+        "fontSize": 12,
+        "fontFamily": "Arial-BoldMT",
+        "color": "white"
+      }
+    },
+    "showCountDownTimerOnEndScreen": true,
+    "countDownTime": "10"
+  },
+  "shareScreen": {
+    "shareContent": ["social", "embed"],
+    "embed": {
+      "source": "<iframe width='640' height='480' frameborder='0' allowfullscreen src='//player.ooyala.com/static/v4/stable/4.10.6/skin-plugin/iframe.html?ec=<ASSET_ID>&pbid=<PLAYER_ID>&pcode=<PUBLISHER_ID>'></iframe>"
+    }
+  },
+  "moreOptionsScreen": {
+    "brightOpacity": 1.0,
+    "darkOpacity": 0.4,
+    "iconSize": 30,
+    "color": "white",
+    "iconStyle": {
+      "active": {
+        "color": "#FFFFFF",
+        "opacity": 1.0
+      },
+      "inactive": {
+        "color": "#FFFFFF",
+        "opacity": 0.95
+      }
+    }
+  },
+  "closedCaptionOptions": {
```

Comment:
I see that https://raw.githubusercontent.com/Wikia/app/1abd8e78f48b70fba8327276b163c7e4413dff91/extensions/wikia/ArticleVideo/scripts/ooyala/html5-skin.js already has a lot of defaults set. We probably can remove most of this config.